dockstore-webservice/src/main/java/io/dockstore/webservice/core/BioWorkflow.java

```diff
@@ -45,6 +44,9 @@
             + "FROM BioWorkflow w LEFT JOIN w.workflowVersions v "
             + "WHERE w.id in (SELECT ue.id FROM User u INNER JOIN u.entries ue where u.id = :userId) "
             + "GROUP BY w.sourceControl, w.organization, w.repository, w.workflowName, w.dbUpdateDate"),
+    @NamedQuery(name = "io.dockstore.webservice.core.BioWorkflow.getEntryLiteVersionsToAggregate", query =
```

Comment:
This query will also return entries that are unpublished. Just checking, is this okay? The endpoint that uses this query is an admin/curator endpoint.

Reply:
Seems ok, opens the question as to whether we should allow adding metrics to unpublished workflows, which also seems ok
(i.e. not super useful, but doesn't seem harmful either)

Reply:
Currently, we can't submit metrics for unpublished workflows, so maybe we should only look at published workflows for now to avoid aggregating metrics for unpublished versions that we can't even post to the webservice

Reply:
I'm ok with that for now, we should consider lifting that restriction on a separate ticket.

Reply:
In theory a workflow could have been run, metrics gathered, then unpublished, then metrics submitted (new ticket is fine).

Reply:
Created https://ucsc-cgl.atlassian.net/browse/SEAB-6650